src/components/TopBar.js

```diff
@@ -313,19 +313,19 @@ const TopBar = ({ showMenu = true, title, href, children }) => {
           div({
             style: { flex: 'none', padding: 28, marginTop: 'auto' }
           }, [
-            h(CromwellVersionLink, { variant: 'light', style: { textDecoration: 'underline', color: colors.accent(0.2) } }),
+            h(CromwellVersionLink, { variant: 'light', style: { textDecoration: 'underline', color: colors.light() } }),
             isBioDataCatalyst() && h(Fragment, [
               h(Link,
                 {
                   variant: 'light',
-                  style: { display: 'block', textDecoration: 'underline', color: colors.accent(0.2) },
+                  style: { display: 'block', textDecoration: 'underline', color: colors.light() },
```

Comment:
Changed for consistency, as above.

Reply:
![image](https://user-images.githubusercontent.com/484484/157467173-a867e4cf-9d53-4cf5-a2ed-67b8058e243d.png)